gradle-plugin/build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
 
     testImplementation(
             gradleTestKit(),
+            "com.android.tools:common:30.0.4"
```

Comment:
to use handy https://cs.android.com/android-studio/platform/tools/base/+/mirror-goog-studio-main:common/src/main/java/com/android/Version.java?q=ANDROID_GRADLE_PLUGIN_VERSION